Change Yoast detected Notice to AIOSEOP_Notices class

EkoJR opened the pull request:
Issue #1853

arnaudbroes commented:
@EkoJR 

> **Plugin could not be activated because it triggered a fatal error.**
> Fatal error: Uncaught Error: Call to undefined function aioseop_disable_notice_yoast_detected() in /app/public/wp-content/plugins/all-in-one-seo-pack-9e236cf5f5bc98964cc322d8bb5b8d96f30f5c7d/all_in_one_seo_pack.php:270 Stack trace: #0 /app/public/wp-content/plugins/all-in-one-seo-pack-9e236cf5f5bc98964cc322d8bb5b8d96f30f5c7d/all_in_one_seo_pack.php(504): aioseop_activate() #1 /app/public/wp-includes/class-wp-hook.php(286): aioseop_install('') #2 /app/public/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array) #3 /app/public/wp-includes/plugin.php(465): WP_Hook->do_action(Array) #4 /app/public/wp-admin/plugins.php(177): do_action('activate_all-in...') #5 {main} thrown in /app/public/wp-content/plugins/all-in-one-seo-pack-9e236cf5f5bc98964cc322d8bb5b8d96f30f5c7d/all_in_one_seo_pack.php on line 270

EkoJR commented:
I have the error fixed. 

However, it appears Yoast made some changes since the last time this was worked on. The class `WPSEO_Import_AIOSEO_Hooks` in `admin/meta_import.php` extends to a class ( `WPSEO_Import_Hooks` ) that no longer exists. Which occured on Yoast 7.1 update ([git commits](https://github.com/Yoast/wordpress-seo/commits/7.1)); [version 7.0.3](https://github.com/Yoast/wordpress-seo/tree/7.0.3/admin/import) contains the old class. This part may take some figuring out.
